src/encoded/types/tracking_item.py

```diff
@@ -0,0 +1,46 @@
+from typing import Any, Dict, Optional, List
```

Comment:
Will need to add an insert for this item to `workbook-inserts` for tests to pass.

Reply:
@drio18 done: https://github.com/smaht-dac/smaht-portal/blob/c5e6a52a3f61b4bfe6c8af258879ee08cd575f2c/src/encoded/tests/data/workbook-inserts/tracking_item.json